src/Hosting/WindowsServices/src/Microsoft.AspNetCore.Hosting.WindowsServices.csproj

```diff
@@ -15,5 +15,10 @@
   <ItemGroup>
     <Reference Include="System.ServiceProcess.ServiceController" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Undo

Reply:
This happens every time I modify hosting...